### DISCUSSION TITLE
Script: Automatically build and destroy coal mine

### DISCUSSION THREAD

**Discussion by Oviz** [fedorovvl/tso_client#291] [General]
Hello, I'm looking for a script which automatically build and destroy coal or gold mine (if exsist) 
We need a clean map cleared of coal or gold mines

**Comment by skelgaard** [comment]
automation functions are not supported

**Comment by Oviz** [comment]
ok, lets start build 1 and cancel :) , and next, next :)
= 6 coal mines,
Its posible?

**Comment by Oviz** [comment]
Any idea, how to made it?

**Comment by skelgaard** [comment]
there is no auto build building and there newer will be.... no automations is supported...
do your own client if you want that
